huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_desktop_pool_action_test.go

```diff
@@ -0,0 +1,103 @@
+package workspace
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDesktopPoolAction_basic(t *testing.T) {
+	var (
+		name         = acceptance.RandomAccResourceNameWithDash()
+		resourceName = "huaweicloud_workspace_desktop_pool_action.test"
+	)
+
+	resource.Test(t, resource.TestCase{
```

Comment:
```suggestion
	resource.ParallelTest(t, resource.TestCase{
```
Using `ParallelTest` means running acceptance tests in parallel, `test` is executed serially.

Reply:
done